pkg/driver/camera/camera_linux.go

```diff
@@ -243,6 +260,7 @@ func (c *camera) VideoRecord(p prop.Media) (video.Reader, error) {
 				// Camera has been stopped.
 				return nil, func() {}, err
 			}
+			c.prevFrameTime = time.Now()
```

Comment:
let's do this if `			if p.DiscardFramesOlderThan != 0 {`

Reply:
Done.